### ISSUE TITLE
List does not show up in "code view"

### ISSUE THREAD

**Issue by lgrn** [sveale/remote-edit#91]
This is on 1.8.10.

When I'm in the "coding view" (editing a file), the Ctrl+Alt+b shortcut renders no result. If I close it (the coding view, with Ctrl+W), it works again when Atom has no open tabs. Is this expected behavior?

Since the Ctrl+Shift+p shortcut works even in the "coding view" I am suspecting this is a bug.

**Comment by sveale** [comment]
This is not expected behavior. 

All of this is handled by Atom itself through the keymap file so I'm not really sure why it's happening.

**Comment by jlami** [comment]
I believe this is because Atom has a default mapping for Ctrl+Alt+b now, jumping the cursor back to the previous subword boundary.

**Comment by sveale** [comment]
Makes sense. I'm on Mac OS X so I don't have the ability to test this myself.

Suggestions for new mappings for windows would be greatly appreciated :)

**Comment by jlami** [comment]
I'll have to look into that, just starting with Atom.
It does show me one thing now, there are way to many key bindings by default ;)

Maybe a multi key binding? "Ctrl-o f", though I don't know if this will overwrite Ctr-o

**Comment by sveale** [comment]
lol thats probably true.
I'll have a think about it.

@lgrn a temporary solution until new key mappings have been added and published would be to trigger browse from the command palette.
